React/src/interfaces/userData.ts

```diff
@@ -0,0 +1,8 @@
+export interface UserData {
```

Comment:
Try to use meaningful names, `UserData` doesn't explain anything.
A good name could be:
`RegisterUser`